shell/cgo-enabled.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# This is a small script to determine whether a service manifest has
+# explicitly enabled compiling with CGo, and print out the appropriate
+# value for the CGO_ENABLED environment variable. Defaults to disabled.
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# shellcheck source=./lib/bootstrap.sh
+source "$DIR/lib/bootstrap.sh"
+
+if [[ "$(yq -r ".arguments.enableCGo" <"$(get_service_yaml)")" == "true" ]]; then
```

Comment:
Add field to `manifest.yaml`?

Reply:
I swear I did that but I'm not seeing it anymore.